test/e2e/infra/vagrant/playbook/roles/common/tasks/busybox.yml

```diff
@@ -1,11 +1,2 @@
 - name: Pull busybox:latest docker image (Ansible < '2.8.0')
-  docker_image:
-    name: busybox:latest
-    pull: yes
-  when: ansible_version.full is version_compare('2.8.0', '<')
-
-- name: Pull busybox:latest docker image (Ansible >= '2.8.0')
-  docker_image:
-    name: busybox:latest
-    source: pull
-  when: ansible_version.full is version_compare('2.8.0', '>=')
+  ansible.builtin.command: ctr -n=k8s.io image pull docker.io/library/busybox:latest
```

Comment:
please use:
```
diff --git a/test/e2e/infra/vagrant/playbook/roles/common/tasks/busybox.yml b/test/e2e/infra/vagrant/playbook/roles/common/tasks/busybox.yml
index 87b7293c..2da2cec0 100644
--- a/test/e2e/infra/vagrant/playbook/roles/common/tasks/busybox.yml
+++ b/test/e2e/infra/vagrant/playbook/roles/common/tasks/busybox.yml
@@ -1,2 +1,9 @@
-- name: Pull busybox:latest docker image (Ansible < '2.8.0')
-  ansible.builtin.command: ctr -n=k8s.io image pull docker.io/library/busybox:latest
+- name: Pull busybox:latest docker image
+  shell: |
+    ctr -n=k8s.io images check name==docker.io/library/busybox:latest > /tmp/busybox1
+    ctr -n=k8s.io image pull docker.io/library/busybox:latest
+    ctr -n=k8s.io images check name==docker.io/library/busybox:latest > /tmp/busybox2
+    diff /tmp/busybox1 /tmp/busybox2 || echo "Image updated"
+    rm -f /tmp/busybox1 /tmp/busybox2
+  register: busybox
+  changed_when: "'Image updated' in busybox.stdout"
```
this will capture whether are not the step caused a change